website/docs/components/pagination/partials/code/how-to-use.md

```diff
@@ -1,22 +1,22 @@
-## "Compact" vs "Numbered" pagination
+## “Compact” vs “Numbered” Pagination
 
 There are two different variants of the `Pagination` component (with different ways to invoke them) built to cover different use cases, contexts, and designs you may need them for.
 
-This differentiation is necessary to cover both use cases of a pagination for a list with a known number of elements (**"numbered"**) and one in which this information is not available or is [cursor-based](https://jsonapi.org/profiles/ethanresnick/cursor-pagination/) (**"compact"**).
+This differentiation is necessary to cover both use cases of pagination for a list with a known number of elements (**"numbered"**) and one in which this information is not available or is [cursor-based](https://jsonapi.org/profiles/ethanresnick/cursor-pagination/) (**"compact"**).
```

Comment:
we seem be to be using a mix of `Compact`, `“Compact”`, and `**"compact"**` (same for Numbered) and we can potentially try to settle on one form

Reply:
Hmm, let me look more closely and think about this one...

Reply:
Ok, I decided to get rid of quotes around Numbered vs Compact when it was referencing the components and fix casing.